Cargo.toml

```diff
@@ -13,7 +13,18 @@ edition = "2018"
 autotests = true
 
 [package.metadata.docs.rs]
-all-features = true
+features = [
+    "blocking",
+    "cookies",
+    "json",
+    "multipart",
+    "stream",
+    "trust-dns",
+    "socks",
+    "gzip",
+    "brotli",
+    "deflate",
+]
```

Comment:
Instead of this, we could add to `--cfg` below, right?

Reply:
Yes, we can add `--cfg reqwest_unstable` below.

Reply:
Done